server/http.go

```diff
@@ -25,12 +25,14 @@ import (
 	"net/http"
 
 	"github.com/improbable-eng/grpc-web/go/grpcweb"
+	"github.com/onflow/flow-emulator/server/backend"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 const (
-	LivenessPath = "/live"
-	MetricsPath  = "/metrics"
+	LivenessPath    = "/live"
+	MetricsPath     = "/metrics"
+	EmulatorApiPath = "/api/"
```

Comment:
Would it make more sense for this to be `/emulator` since the API is vague and then you would have `/emulator/snapshot`. Do you have some reasons?